CSharp/Concentus/Concentus.targets

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <_ConcentusNativeBinaries Include="$(MSBuildThisFileDirectory)..\content\**\*.dll" />
```

Comment:
Wouldn't it be easier to just package the binaries under `runtimes/...` in the package so NuGet can take care of all this automatically?